android/titanium/src/java/org/appcelerator/titanium/proxy/TiViewProxy.java

```diff
@@ -552,6 +552,25 @@ public void releaseViews()
 	 */
 	public abstract TiUIView createView(Activity activity);
 
+	
+	/**
+	 * Adds multiple child to this view proxy.
+	 * @param childs The child view proxies to add.
+	 * @module.api
+	 */
+	@Kroll.method
+	public void add(TiViewProxy[] childs)
```

Comment:
We should be using the same add method as below in line 580 (originally 561). Instead, we change the params to allow for an Object class, and in the method, we check the type if it is of type TiViewProxy child or if it's some form of array, cast it correctly and handle the code appropriately.